### ISSUE TITLE
UI elements overlap at high display scaling factors

### ISSUE THREAD

**Issue by WSHAPER** [leandrosa81/taskbar-monitor#56]
### TL;DR
When using a high-resolution display with Windows scaling enabled, the taskbar-monitor widget's UI elements overlap, making it difficult to read and use. This issue occurs on a 3840x2400 display with 250% scaling enabled.

### Steps to reproduce:
1. Install taskbar-monitor on a system with a high-resolution display (e.g., 3840x2400)
2. Set Windows display scaling to a high value (e.g., 250%)
3. Observe the taskbar-monitor widget

### Current behavior:
The UI elements of the widget overlap each other, making the information unreadable. CPU, MEM, DISK, and NET labels are stacked on top of each other, and the usage bars are not properly aligned.

### Expected behavior:
The widget should scale appropriately, maintaining readability and proper layout at all display scaling factors.

### Screenshots:
![image](https://github.com/user-attachments/assets/015473fc-44be-43e6-8207-2f8e132a4d7b)

### Tested Environment:
- Windows version: [Windows 10, version 21H2](https://en.wikipedia.org/wiki/Windows_10,_version_21H2)
- NET Framework Setup
    ```powershell
    reg query "HKLM\SOFTWARE\Microsoft\NET Framework Setup\NDP\v4\Full" /v Release
    ```
    Returns
    ```powershell
    HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\NET Framework Setup\NDP\v4\Full
        Release    REG_DWORD    0x8234d
    ```
    533325 (`0x8234d`) is greater than 533320, which means I must have .NET Framework 4.8.1 installed on my system

- Display resolution: 3840x2400
- Display scaling: 250%
- taskbar-monitor version: v0.4.2

### Question:
Are there any current workarounds or configuration options that might help address this issue in the short term?

I'd be happy to provide any additional information or assist with testing if needed.

**Comment by fAlkon35** [comment]
I have the same problem.

Tested Environment:

PC: Surface Pro7

Windows version: Windows 11, version 24H2

NET Framework Installation

Screen resolution: 2736x1824

Screen scaling: 150%

Taskbar monitor version: v0.4.2

![2024-11-02 21 43 17](https://github.com/user-attachments/assets/2fff1403-2e93-45c6-b3f6-f2183cdc2bd6)
